papaparse.js

```diff
@@ -969,7 +987,7 @@
 
 		function parseDynamic(field, value)
 		{
-			if ((_config.dynamicTyping[field] || _config.dynamicTyping) === true)
+			if (_config.dynamicTyping.isIncluded(field))
```

Comment:
The only time I think we need to worry about caching is if the user provides a slow callback; still, that's something they can cache themselves.